go.mod

```diff
@@ -3,17 +3,18 @@ module github.com/openshift/windows-machine-config-operator
 go 1.16
 
 replace (
-	sigs.k8s.io/cluster-api-provider-aws => github.com/openshift/cluster-api-provider-aws v0.2.1-0.20200520125206-5e266b553d8e // This is coming from machine-api repo
-	sigs.k8s.io/cluster-api-provider-azure => github.com/openshift/cluster-api-provider-azure v0.1.0-alpha.3.0.20200902180535-72169c58a81f
+	// These are coming from the machine-api repo
+	sigs.k8s.io/cluster-api-provider-aws => github.com/openshift/cluster-api-provider-aws v0.2.1-0.20210622023641-c69a3acaee27
+	sigs.k8s.io/cluster-api-provider-azure => github.com/openshift/cluster-api-provider-azure v0.1.0-alpha.3.0.20210816141152-a7c40345b994
```

Comment:
Just FYI, we are trying to get rid of this issue during the 4.10 cycle. The problem is that MAPI imports types from these repos currently, we will migrate them to openshfit/api to avoid everyone having to vendor this like this